mkfontscale.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: mkfontscale
+  version: 1.2.2
+  epoch: 0
+  description: Scalable font index generator for X
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: MIT
+  scriptlets:
+    trigger:
+      paths:
+        - /usr/share/fonts/*
+      script: |
+          #!/bin/sh
+          for i in "$@"; do
+              mkfontscale "$i"
```

Comment:
This one should also call `mkfontdir "$i"`.

Reply:
good catch thanks